src/gt4py/next/otf/compilation/build_systems/compiledb.py

```diff
@@ -67,15 +69,13 @@ def __call__(
             implicit_domain=source.program_source.implicit_domain,
         )
 
-        if self.renew_compiledb or not (
-            compiledb_template := _cc_find_compiledb(cc_prototype_program_source, cache_lifetime)
-        ):
-            compiledb_template = _cc_create_compiledb(
-                cc_prototype_program_source,
-                build_type=self.cmake_build_type,
-                cmake_flags=self.cmake_extra_flags or [],
-                cache_lifetime=cache_lifetime,
-            )
+        compiledb_template = _cc_get_compiledb(
```

Comment:
Just a  question, and probably out of scope of this PR: why is this variable called `compiledb_template`? Wouldn't it be more appropriate something like `compiledb_path`?

Reply:
I think that's driven by how it is used. The stuff in the compiledb directory is used as a template for the other programs (it does a `copytree`)...